pulsar-client-cpp/include/pulsar/c/result.h

```diff
@@ -53,14 +53,15 @@ typedef enum {
 
     pulsar_result_ConsumerNotInitialized,         /// Consumer is not initialized
     pulsar_result_ProducerNotInitialized,         /// Producer is not initialized
+    pulsar_result_ProducerBusy,                   /// Producer with same name is already connected
```

Comment:
Ah, it's a mess, the error code returned from the C-API was wrong anyway, because this wasn't kept up to date.
The root cause was adding it in the middle for https://github.com/apache/pulsar/pull/3297. We need to stop this from happening again.